src/odemis/util/peak.py

```diff
@@ -318,14 +372,96 @@ def estimateFitTime(self, data):
         return len(data) * 10e-3  # s
 
 
-def Curve(wavelength, peak_parameters, offset, type='gaussian'):
+def peak_to_energy(pos, width, amplitude):
+    """
+    Converts the peaks to energy domain.
+    Args:
+        pos: the center position of the peak in 'm'
+        width: the width of the peak in 'm'
+        amplitude: the amplitude of the peak in space domain
+
+    Returns: peak parameters as (pos, width, amplitude) in energy domain
+
+    """
+    amplitude_e = (amplitude * pos ** 2) / WL_TO_ENERGY
```

Comment:
If you sort them in the same order as they are defined in the function signature, it would be a little bit easier to read.